src/gen9_avc_encoder.c

```diff
@@ -1038,6 +1174,7 @@ gen9_avc_allocate_resources(VADriverContextP ctx,
                                                    "sfd output buffer");
         if (!allocate_flag)
             goto failed_allocation;
+        i965_zero_gpe_resource(&avc_ctx->res_sfd_output_buffer);
```

Comment:
Are the two "i965_zero_gpe_resource" necessary? it affect the performance as memory map operation.

Reply:
if avc_ctx->res_sfd_output_buffer  is not set to zeros, It will cause inconsistent performance.